tests/multipart.cpp

```diff
@@ -178,4 +178,39 @@ TEST_CASE("multipart legacy test", "[multipart]")
     assert(received.empty());
     assert(str == "One-hundred");
 }
+
+TEST_CASE("multipart codec", "[multipart]")
+{
+    using namespace zmq;
+
+    multipart_t mmsg;
+    mmsg.addstr("Hello World");
+    message_t msg = mmsg.encode();
+    assert(msg.size() == 1 + 11); // small size packing
```

Comment:
The CHECK or REQUIRE macro should be used for unit tests.

Looks like the older unit tests in the file need a bit a refactoring.

Reply:
I'll leave the older test to others but I will re-instrument this test with macros as suggested.